invenio_rdm_records/searchconfig.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
```

Comment:
This is reused in App-RDM for the RSK app config generation to ensure there's a coherent way of selecting among available options.